Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Use an official Python runtime as a parent image
+FROM python:3
+
+# Set the working directory to /app
+WORKDIR /app
+
+# Copy the current directory contents into the container at /app
+COPY . /app
+
+# Install any needed packages specified in requirements.txt
+RUN pip3 install --upgrade lastcast
```

Comment:
I'm thinking this should probably be running `python setup.py install` instead, otherwise I don't think there's a reason for the `COPY . /app` line?

Reply:
I am certainty not a Docker expert but I believe that I at do need to run `pip3 install --upgrade lastcast` in order to build the dependencies in the container. I could copy only `setup.py` and then run `pip3 install --upgrade lastcast`

Or I could copy over the whole directory `COPY . /app` and then run `python setup.py install`. (If you do not copy the working directory, than the `lastcast` dir/package is not found during the setup.

Let me know which you would prefer. I updated the image to only copy over `setup.py` and then run the pip3 command.

Reply:
I'm thinking `COPY . /app` + `python setup.py install` would be better, since it means you could iterate on the dockerfile without depending on the latest version of `lastcast` pushed to pypi/

Reply:
Oh of course, that makes sense! I had literally just taken the Linux setup steps and translated them to Dockerfile.